src/types/value-inspector.interface.ts

```diff
@@ -1,11 +1,8 @@
-import { ClassReflection } from '@plumier/reflect';
 import { PropertyDto } from './property-dto.interface';
 import { IClassProcessor } from '../types/iclass-processor.interface';
 
 export interface ValueInspector {
   shouldInspect(propertyDto: PropertyDto): boolean;
 
-  hasCircularClassFixture(parentClassReflection: ClassReflection, propertyDto: PropertyDto): boolean;
```

Comment:
reflect-metadata will not protect you from all circular references. you still need this check. please don't remove